vars/capsuleSystemTests.groovy

```diff
@@ -0,0 +1,196 @@
+/* groovylint-disable 
+  BuilderMethodWithSideEffects, CompileStatic, DuplicateStringLiteral, 
+  FactoryMethodName, VariableTypeRequired */
+
+
+void gitClone(String repository, String branch, String directory, String credentialsId) {
```

Comment:
maybe lets move gitClone to `vars/gitClone.groovy` and call it as standard method? We copy this piece of code between multiple pipelines

Reply:
Will do that as separated PR, and let's merge this pr after the git clone one, so i can use it :)